File: Backend/src/main/java/com/sparklead/sharaspot/repositories/ChargingStationRepository.java
package com.sparklead.sharaspot.repositories;

import com.sparklead.sharaspot.models.ChargingStation;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ChargingStationRepository extends MongoRepository<ChargingStation,Integer> {

}

File: app/src/main/java/com/sparklead/sharaspot/ui/repositories/ChargingStationRepository.kt
package com.sparklead.sharaspot.ui.repositories

import com.sparklead.sharaspot.models.ChargingStationResponse
import kotlinx.coroutines.flow.Flow

interface ChargingStationRepository {

    fun getChargingStationList(): Flow<ArrayList<ChargingStationResponse>>

}